crates/sui-graphql-rpc/src/types/balance.rs

```diff
@@ -142,7 +157,17 @@ impl RawPaginated<Cursor> for StoredBalance {
 
 impl Target<Cursor> for StoredBalance {
     fn cursor(&self) -> Cursor {
-        Cursor::new(self.coin_type.clone())
+        Cursor::new(BalanceCursor {
+            coin_type: self.coin_type.clone(),
+            checkpoint_viewed_at: None,
+        })
```

Comment:
Do we expect this path to get used anymore? If not, should we `panic!` in it to make sure it doesn't get used?

Reply:
Handled at https://github.com/MystenLabs/sui/pull/15996